samples/MqttSample/MqttSample.csproj

```diff
@@ -5,34 +5,18 @@
     <TargetFramework>net5.0</TargetFramework>
     <LangVersion>latest</LangVersion>
     <AssemblyName>OMP.Connector.EdgeModule</AssemblyName>
-    <RootNamespace>OMP.Connector.EdgeModule</RootNamespace>
-    <VersionPrefix>5.0.0</VersionPrefix>
-    <Configurations>DebugKafka;Release;DebugMqtt</Configurations>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='DebugKafka|netcoreapp3.1|AnyCPU'">
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-    <TreatSpecificWarningsAsErrors />
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='DebugMqtt|netcoreapp3.1|AnyCPU'">
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-    <TreatSpecificWarningsAsErrors />
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='DebugKafka|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;DEBUGKAFKA</DefineConstants>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='DebugMqtt|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;DEBUGMQTT</DefineConstants>
+    <RootNamespace>OMP.Connector.EdgeModule</RootNamespace>    
+    <Configurations>Release;Debug</Configurations>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
```

Comment:
What is the purpose of these docker properties?
Only asking for educational purposes ;-)

Reply:
This setting is automatically added by Visual Studio when enabling docker support. So I think it is more related to VS-Docker integration when running the image directly from VS. But I assume it has no real baring on 'native' docker